miio/integrations/roborock/vacuum/vacuum.py

```diff
@@ -74,6 +74,7 @@
 ROCKROBO_T7SPLUS = "roborock.vacuum.a23"
 ROCKROBO_S7_MAXV = "roborock.vacuum.a27"
 ROCKROBO_S7_PRO_ULTRA = "roborock.vacuum.a62"
+ROCKROBO_S8_PRO_ULTRA = "roborock.vacuum.a70"
```

Comment:
you can also add it to  `AUTO_EMPTY_MODELS` see line 120

Reply:
Absolutely, thanks for pointing that out!